pebblo/app/service/classification.py

```diff
@@ -106,6 +101,15 @@ def process_request(self):
                 return PebbloJsonResponse.build(
                     body={"error": "Input data is missing"}, status_code=400
                 )
+            if not req.mode:
+                req.mode = config_details.get("classifier", {}).get(
+                    "mode", ClassificationMode.ALL.value
+                )
+                req.mode = ClassificationMode(req.mode)
+            if not req.anonymize:
+                req.anonymize = config_details.get("classifier", {}).get(
+                    "anonymizeSnippets", False
+                )
```

Comment:
Same comment as above. Let's honor the `mode` and `anonymize` args as the client requested.